logbat_view/src/main/java/info/logbat_view/common/util/UUIDConvertor.java

```diff
@@ -0,0 +1,24 @@
+package info.logbat_view.common.util;
+
+import java.util.UUID;
+import java.util.regex.Pattern;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class UUIDConvertor {
+
+    private static final Pattern UUID_PATTERN = Pattern.compile(
+        "^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$");
+
+    public static UUID convertStringToUUID(String uuid) {
+        if (uuid == null || uuid.isBlank()) {
+            throw new IllegalArgumentException("UUID는 필수 값 입니다.");
+        }
+        if (!UUID_PATTERN.matcher(uuid).matches()) {
+            throw new IllegalArgumentException("UUID 형식이 올바르지 않습니다.");
+        }
+        return UUID.fromString(uuid);
```

Comment:
`UUID.fromString()` 를 통해 생성될때 내부적으로 검증을 하고, 올바른 형식이 아닐경우 `IllegalArgumentException` 을 던지기때문에 직접 패턴으로 검증하기보다 이렇게 해보는건 어떨가요?

```suggestion
        try {
            return UUID.fromString(uuid);
        }
        catch (IllegalArgumentException e) {
            throw new IllegalArgumentException("UUID 형식이 올바르지 않습니다.");
        }
```

Reply:
`IllegalArgumentException` 을 캐치해서 `IllegalArgumentException`를 던진다는 로직이 뭔가 이상한거 같아서 배제했었는데 의견이 궁금합니다! 단순 에러 메시지 변경으로만 넘기는 것이 적절할까요?!

Reply:
엇 그렇네요!
제가 무의식적으로 전역 에러핸들러가 생길것이고 거기서 메세지를 활용할거라 생각하고 있었나봐요.
이후 에러 처리에 대한 이야기를 정리하는게 좋겠네요!